src/Tools/dotnet-monitor/IDiagnosticLifetimeService.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.Diagnostics.Tools.Monitor
     /// <summary>
     /// Callbacks for the lifetime events related to the diagnostic connection.
     /// </summary>
-    internal interface IDiagnosticLifetimeService
```

Comment:
Is this the correct assembly for this type? Having references to the tool directly now may make this more complicated to refactor to in the future.

Reply:
We have a couple types here that we need to expose from this assembly (also a new one that will get added in a follow-up PR). 

Similar to my other comment: We aren't very consistent with where types live so things are pretty spread out between assemblies currently. I'd prefer that any refactoring around where types live be a separate, more conscious, effort to ensure it makes sense. We aren't locked into any of the current assemblies where public types are in -- we can refactor whenever we like without issue